pkg/client/cache/expiration_cache.go

```diff
@@ -191,6 +199,12 @@ func (c *ExpirationCache) Replace(list []interface{}, resourceVersion string) er
 	return nil
 }
 
+// Resync will touch all objects to put them into the processing queue
+func (c *ExpirationCache) Resync() error {
+	// Nothing to do
+	return nil
```

Comment:
I think this is incorrect.
What we should do here, is to call "Resync()" of the underlying cacheStorage.